js/start.js

```diff
@@ -404,6 +404,45 @@ function onboardIfNeeded() {
   return onboardIfNeededDeferred.promise();
 }
 
+function addDoorbell() {
```

Comment:
Can we add this to the startup folder (either in the index file or it's own file)? That folder was created so that start.js doesn't get ridiculously huge and confusing with all these little loose ends.